### PR TITLE
Bug 1842229 - APK creds cleanup and add stage autograph

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Depends on https://github.com/mozilla-services/cloudops-infra/pull/5012

### CODE REVIEW

File: signingscript/docker.d/passwords.yml [hunk 1]
```diff
@@ -97,6 +97,16 @@ in:
...
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_FOCUS_STAGE_USERNAME"},
+             {"$eval": "AUTOGRAPH_FOCUS_STAGE_PASSWORD"},
+             ["autograph_focus_stage"]
+            ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_FENIX_STAGE_USERNAME"},
+             {"$eval": "AUTOGRAPH_FENIX_STAGE_PASSWORD"},
+             ["autograph_apk_stage", "autograph_apk_mozillaonline_stage"]
+            ]
```
Comment (ahal) [inline signingscript/docker.d/passwords.yml:109]: I think the plan was to set this up against adhoc right?

**Review by ahal — CHANGES_REQUESTED** [review]

File: signingscript/docker.d/passwords.yml (hunk 1 above)
Comment (ahal) [reply signingscript/docker.d/passwords.yml:109]: I recall our conversation about setting it up in both places now, but you'll still need to add this for adhoc.

Comment (JohanLorenzo) [reply signingscript/docker.d/passwords.yml:109]: My bad, I though adhoc signing was covered in this block but I realized it has its own section below. Done!

File: signingscript/docker.d/init_worker.sh [hunk 2]
```diff
@@ -113,10 +111,14 @@ case $ENV in
       mobile)
         test_var_set 'AUTOGRAPH_FENIX_PASSWORD'
         test_var_set 'AUTOGRAPH_FENIX_USERNAME'
+        test_var_set 'AUTOGRAPH_FENIX_STAGE_PASSWORD'
+        test_var_set 'AUTOGRAPH_FENIX_STAGE_USERNAME'
```
Comment (ahal) [inline signingscript/docker.d/init_worker.sh]: This will need to be changed to `AUTOGRAPH_STAGE_FENIX_PASSWORD` etc as well now.

**Review by ahal — CHANGES_REQUESTED** [review]
Thanks!

There are some test failures. The `test_init.py` ones are because you need to add the new variables to:
https://github.com/mozilla-releng/scriptworker-scripts/blob/master/tests/test_init.py#L79

**Comment by JohanLorenzo** [comment]
All test failures are now fixed!

**Comment by JohanLorenzo** [comment]
New creds were tested in https://github.com/mozilla-mobile/firefox-android/pull/2886 and more precisely with these tasks: 

* https://firefox-ci-tc.services.mozilla.com/tasks/ZDlCZbbTSl-MD-UUc8MZjg
* https://firefox-ci-tc.services.mozilla.com/tasks/UR7yWcAyRTOcs_T9TU8gTg

I think we're good to proceed get this merged. Let's put this in today to get signing_script deployment stabilized. r? @gabrielBusta

**Review by gabrielbusta — APPROVED** [review]

**Comment by gabrielbusta** [comment]
Thanks for testing it

**Pull request merged**